app/classifier/tasks/survey/chooser/Choices.jsx

```diff
@@ -45,9 +45,16 @@ class Choices extends React.Component {
   }
 
   whatSizeThumbnails({ length }) {
-    if (this.props.task.alwaysShowThumbnails) {
+    // if new survey task thumbnails property is undefined and legacy alwaysShowThumbnails is true, then show thumbnails to support legacy alwaysShowThumbnails functionality
```

Comment:
```suggestion
    // if new survey task.thumbnails property is undefined and legacy task.alwaysShowThumbnails is true, then show thumbnails to support legacy alwaysShowThumbnails functionality
```

Suggestion for clarity